config/overlays/mutation/kustomization.yaml

```diff
@@ -16,6 +16,13 @@ resources:
 - ../../crd/bases/mutations.gatekeeper.sh_assignmetadata.yaml
 - ../../crd/bases/status.gatekeeper.sh_mutatorpodstatuses.yaml
 
+patches:
+- target:
+    group: apiextensions.k8s.io
+    version: v1
+    kind: CustomResourceDefinition
+  path: ../../crd/patches/preserve_unknown_fields_false.yaml
```

Comment:
Looks like the mutation crds in the helm chart did not get updated by kustomize patch.
This patch should be in `config/overlays/mutation_webhook/kustomization.yaml` instead since `cmd/build/helmify/kustomization.yaml` looks at `config/overlays/mutation_webhook`

Reply:
Fixed!